### PR TITLE
NXP-21384: release nuxeo-groups-rights-audit for 9.10

### PR DESCRIPTION
**Pull request by driverpt**

### CODE REVIEW

**Comment by pgmillon** [comment]
👍

File: pom.xml [hunk 1]
```diff
@@ -3,14 +3,14 @@
   <parent>
     <groupId>org.nuxeo.ecm.distribution</groupId>
     <artifactId>nuxeo-distribution</artifactId>
-    <version>8.10-SNAPSHOT</version>
+    <version>9.10-SNAPSHOT</version>
```
Comment (jcarsique) [inline pom.xml]: `9.10-HF01-SNAPSHOT`

File: pom.xml [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
   </parent>
 
   <groupId>org.nuxeo.marketplace.groups.rights.audit</groupId>
   <artifactId>parent</artifactId>
   <packaging>pom</packaging>
   <name>Nuxeo Groups Rights Audit Marketplace package</name>
-  <version>1.8.3-SNAPSHOT</version>
+  <version>1.9.0-SNAPSHOT</version>
```
Comment (jcarsique) [inline pom.xml:13]: `.0` is optional but ok

**Review by jcarsique — CHANGES_REQUESTED** [review]
:+1: after version change

File: pom.xml (hunk 2 above)
Comment (driverpt) [reply pom.xml:13]: i like to leave .0 there for consistency.

**Comment by jcarsique** [comment]
:+1:

**Pull request merged**